contrib/freshmaker-cli

```diff
@@ -18,14 +18,19 @@ Example usage:
         ./freshmaker-cli --redhat --stage --name httpd24-docker
     - Search for events in particular state
         ./freshmaker-cli --redhat --stage --state skipped
+    - Search if freshmaker reacted on all  RHSA, in SHIPPED_LIVE state that are IMPORTANT or CRITICAL
```

Comment:
I guess it would be preferable something like "Search for advisories ignored by Freshmaker"